src/plugin_xmms/configure.c

```diff
@@ -46,7 +46,7 @@
 #include "locale_hack.h"
 
 /*
- * Initialize Global Valueable
+ * Initialize Global Valuable
```

Comment:
should this actually be `Variable` ?

Reply:
Can you please change that the "variable"?